docusaurus.config.js

```diff
@@ -32,6 +34,16 @@ module.exports = {
       },
     ],
     "./static/plugins/bpmn-js",
+    [
+      "@docusaurus/plugin-content-docs",
+      {
+        id: "optimize",
+        path: "optimize",
+        routeBasePath: "optimize",
+        beforeDefaultRemarkPlugins: [versionedLinks],
```

Comment:
🔗 This configures the _optimize_ docs to use the MDX plugin that interprets link target URLs based on the source file directory.